src/tactics/FStar.Tactics.Hooks.fst

```diff
@@ -831,27 +837,67 @@ let splice (env:Env.env) (is_typed:bool) (lids:list Ident.lident) (tau:term) (rn
     let gs, sigelts =
       if is_typed then
       begin
-        let e_blob = e_option (e_tuple2 e_string RE.e_term) in
-        let gs, sig_blobs = run_tactic_on_ps tau.pos tau.pos false
-          RE.e_env
-          {env with gamma=[]}
-          (e_list (e_tuple3 e_bool RE.e_sigelt e_blob))
-          tau
-          tactic_already_typed
-          ps 
-        in
-        let sigelts = sig_blobs |> map (fun (checked, se, blob_opt) ->
-          { se with
-              sigmeta = { se.sigmeta with
-                sigmeta_extension_data =
-                      (match blob_opt with
-                       | Some (s, blob) -> [s, Dyn.mkdyn blob]
-                       | None -> []);
-                sigmeta_already_checked = checked; }
-          }
-        )
-        in
-        gs, sigelts
+        //
+        // See if there is a val for the lid
+        //
+        if List.length lids > 1
+        then let s = lids |> List.map Ident.string_of_lid |> BU.concat_l ", " in
```

Comment:
`s` unused? :)

Reply:
Ah yes, thanks! At some point I was adding this in the error message but then I switched to the show method.